webview-ui/src/index.tsx

```diff
@@ -4,11 +4,16 @@ import "./index.css"
 import App from "./App"
 import reportWebVitals from "./reportWebVitals"
 import "../../node_modules/@vscode/codicons/dist/codicon.css"
+import { PostHogProvider } from "posthog-js/react"
 
 const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 root.render(
 	<React.StrictMode>
-		<App />
+		<PostHogProvider
+			apiKey="phc_5WnLHpYyC30Bsb7VSJ6DzcPXZ34JSF08DJLyM7svZ15"
+			options={{ api_host: "https://us.i.posthog.com", capture_pageview: false }}>
```

Comment:
nit: Consider creating a `const` for the url and secret so we know where they are.